recipe/meta.yaml

```diff
@@ -1,45 +1,117 @@
+{% set name = "dask" %}
 {% set version = "0.19.1" %}
 
 package:
-  name: dask
-  version: "{{ version }}"
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aae1f237870c01773bca5a75289f10166b84e13b9e966d8431e945f1f1febb99
 
 build:
-  number: 0
+  number: 1
   noarch: python
 
 requirements:
-  build:
-    - python
-
   run:
-    - python
     - bokeh >=0.13.0
-    - cloudpickle >=0.2.1
-    - cytoolz >=0.7.3
-    - dask-core {{ version }}
-    - distributed >=1.23.1
-    - numpy >=1.11.0
-    - pandas >=0.19.0
-    - partd >=0.3.8
-    - toolz >=0.7.3
+    - {{ pin_subpackage(name|lower + '-core', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-array', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-bag', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-dataframe', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-distributed', exact=True) }}
+    - {{ pin_subpackage(name|lower + '-delayed', exact=True) }}
 
-test:
-  imports:
-    - dask
-    - dask.array
-    - dask.bag
-    - dask.bytes
-    - dask.dataframe
-    - dask.dataframe.tseries
-    - dask.delayed
-    - dask.diagnostics
-    - dask.distributed
-    - distributed
+outputs:
+  - name: {{ name|lower + '-core' }}
+    build:
+      noarch: python
+      # I  really don't know why I can't use {{ PYTHON }}, it just won't rerender :/
+      script: "python -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - python
+      run_constrained:
+        - cloudpickle >=0.2.1
+        - cytoolz >=0.7.3
+        - distributed >=1.23.1
+        - numpy >=1.11.0
```

Comment:
@mrocklin do you want to support people having `dask-core` and numpy<1.11 at the same time? Admitidly, I don't think conda-forge has numpy less than 1.11, but you answer would give guidance on how to handle other minimum version pinnings.